src/main/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtil.java

```diff
@@ -185,4 +192,44 @@ public static boolean shouldConvertFreeformTextToAttachment(UploadSchemaKey sche
         Set<String> fieldsToConvert = SCHEMA_FIELDS_TO_CONVERT.get(schemaKey);
         return fieldsToConvert != null && fieldsToConvert.contains(fieldName);
     }
+
+    /**
+     * Helper method to convert a list of ColumnDefinition to a ColumnModel list.
+     * @param columnDefinitions
+     * @return
+     */
+    public static List<ColumnModel> convertToColumnList(final List<ColumnDefinition> columnDefinitions) {
+        ImmutableList.Builder<ColumnModel> columnListBuilder = ImmutableList.builder();
+
+        for (ColumnDefinition columnDefinition : columnDefinitions) {
+            ColumnModel columnModel = new ColumnModel();
+            columnModel.setName(columnDefinition.getName());
+            columnModel.setColumnType(columnDefinition.getColumnType());
+            columnModel.setMaximumSize(columnDefinition.getMaximumSize());
+            columnListBuilder.add(columnModel);
+        }
+
+        return columnListBuilder.build();
+    }
+
+    public static Map<String, String> getRowValuesFromRecordBasedOnColumnDefinition(final Map<String, String> rowMap, final Item record, final List<ColumnDefinition> columnDefinitions, final String recordId, final ExportTask task) {
+        Map<String, String> retMap = new HashMap<>();
+        retMap.putAll(rowMap);
```

Comment:
Instead of intsantiating a map, copying the contents, and returning the new map, maybe we can have this method update the map in-place and have a void return type. This way, we don't have to instantiate more maps then we need, and it makes the code in SynapseExportHelper easier to reason about. (Right now, we take a map, copy it, and then add all those values back to itself.)